compiler/rustc_codegen_ssa/src/sir.rs

```diff
@@ -903,6 +899,10 @@ impl SirTypes {
             idx
         })
     }
+
+    pub fn get_value(&self, tyid: ykpack::TypeId) -> &ykpack::Ty {
```

Comment:
I'd just call that `get()`.

Reply:
I called it `get_value` in case we might want to have a function that returns the key someday. But I'm happy to change it.